examples/object_detection/android/app/download_models.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+tasks.register('downloadModelFile0', Download) {
```

Comment:
Per a discussion with Tuan and Prianka, it sounds like efficientdet has some issues with the scaling due to the way the model is structures and what MediaPipe/Task Library did out of the box. Since EfficientDet2 looks good, let's cut out the other two models for now and only use the one. You can use this link: https://storage.googleapis.com/ai-edge/interpreter-samples/object_detection/android/efficientdet_lite2.tflite

Reply:
@PaulTR 
It seems that the input and output of the model in the current code and from the above link are different.
so I need time to update code